modules/creating-multiple-ingress-through-single-alb.adoc

```diff
@@ -1,12 +1,12 @@
 // Module included in the following assemblies:
 //
-// * networking/aws_load_balancer_operator/multiple-ingress-through-single-alb.adoc
+// * networking/networking-operators/aws_load_balancer_operator/configuring-aws-load-balancer-operator/multiple-ingress-through-single-alb.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="nw-creating-multiple-ingress-through-single-alb_{context}"]
 = Creating multiple ingress resources through a single AWS Load Balancer
 
-You can route the traffic to multiple ingress resources through a single AWS Load Balancer by using the CLI.
+You can route the traffic to different services with multiple ingress resources that are part of a single domain through a single AWS Load Balancer. Each Ingress resource provides different endpoints of the domain.
```

Comment:
This is just an example spot, but around these topics in general I notice "ingress resource" vs "Ingress resource" vs "`Ingress` resource"; suggest syncing up on one (probably "`Ingress` resource" in most cases).